Neos.FluidAdaptor/Tests/Unit/ViewHelpers/Uri/ActionViewHelperTest.php

```diff
@@ -85,6 +85,7 @@ public function renderCorrectlyPassesAllArgumentsToUriBuilder()
      */
     public function renderThrowsViewHelperExceptionIfUriBuilderThrowsFlowException()
     {
+        $exception = null;
```

Comment:
?

Reply:
was added because of line 96 which uses $exception when its possibly undefined...